src/EA.Weee.Web/Areas/AatfReturn/Controllers/ObligatedValuesCopyPasteController.cs

```diff
@@ -40,10 +40,15 @@ public virtual async Task<ActionResult> Index(Guid returnId, Guid aatfId, Guid s
                     AatfId = aatfId,
                     ReturnId = returnId,
                     OrganisationId = @return.ReturnOperatorData.OrganisationId,
-                    SchemeId = schemeId,
-                    SchemeName = Task.Run(() => cache.FetchSchemePublicInfoBySchemeId(schemeId)).Result.Name,
-                    AatfName = Task.Run(() => cache.FetchAatfData(@return.ReturnOperatorData.OrganisationId, aatfId)).Result.Name
+                    AatfName = Task.Run(() => cache.FetchAatfData(@return.ReturnOperatorData.OrganisationId, aatfId)).Result.Name,
+                    Type = obligatedType
                 };
+
+                if (obligatedType == ObligatedType.Recieved)
+                {
+                    viewModel.SchemeId = schemeId;
```

Comment:
Add unit tests to test these properties for when ObligatedType.Received

Reply:
The existing test tested with the scheme data, so I added a unit test to test for when its reused and made sure the scheme properties on the VM were not set correctly.